cg-producer/pypi-readme.md

```diff
@@ -0,0 +1,23 @@
+Library for generating PyPI Call Graphs through PyCG
+====================================================
+
+This library can be used to produce the call graph of a package distributed through pip along with saving its
+source code and the call graph in json format in a specified directory.
+
+Usage
+-----
+
+Example usage after installing the specific package through ```pip```:
```

Comment:
For inline code listings a single ` is enough.

Reply:
Fixed on [01f7959](https://github.com/fasten-project/pypi-tools/pull/6/commits/01f79591f34c0281a9104a8f12e0475b5f9a5234)